2018/2/solution2.py

```diff
@@ -0,0 +1,28 @@
+def get_checksum(box_ids):
+    return (len([box_id for box_id in box_ids if contains_matching(2, box_id)])
+            * len([box_id for box_id in box_ids if contains_matching(3, box_id)]))
+
+
+def contains_matching(n, box_id):
```

Comment:
I really like how all of this came together. It is far cleaner than mine and far more readable